app/assets/javascripts/oxalis/model/binary/data_cube.js

```diff
@@ -194,6 +203,9 @@ class DataCube {
     const cube = this.cubes[zoomStep];
     if (cube != null) {
       const { boundary } = cube;
+      if (x >= boundary[0] || y >= boundary[1] || z >= boundary[2] || x < 0 || y < 0 || z < 0) {
```

Comment:
Oh yes, I didn't see the comment above. The check became necessary when I temporarily disabled boundary checks globally and the calculated indices weren't unique anymore. That was a hard-to-find bug which is why I added the sanity check here. However, I'll re-remove the check so that performance is not harmed :)